app/code/Magento/Sales/Test/Mftf/Test/AdminAvailabilityCreditMemoWithNoPaymentTest.xml

```diff
@@ -61,8 +61,10 @@
         <click selector="{{AdminOrderFormItemsSection.updateItemsAndQuantities}}" stepKey="clickUpdateItemsAndQuantitiesButton"/>
 
         <!--Fill customer group and customer email-->
-        <selectOption selector="{{AdminOrderFormAccountSection.group}}" userInput="{{GeneralCustomerGroup.code}}" stepKey="selectCustomerGroup"/>
-        <fillField selector="{{AdminOrderFormAccountSection.email}}" userInput="{{Simple_US_Customer.email}}" stepKey="fillCustomerEmail"/>
+        <comment userInput="Fill Account Information" stepKey="selectCustomerGroup"/>
```

Comment:
Hi @AnnaAPak. Good job, but I would ask you to highlight that he comment was added for preserving backward compatibility. Something like the following one

```suggestion
        <comment userInput="Comment is added to preserve the step key for backward compatibility" stepKey="selectCustomerGroup"/>
```

Otherwise this comment might be removed as "unnecessary" in the future and BC will be broken. 

Thanks.

Reply:
I have updated the comments as per the recommendation. Thank you.